app/views/asset/show_hwdetails.scala.html

```diff
@@ -26,10 +26,15 @@
         <tr>
           <td>Vendor</td><td>@aa.lshw.base.vendor</td>
         </tr>
-
         <tr>
             <td>Serial</td><td>@aa.lshw.base.serial</td>
         </tr>
+        <tr>
+            <td>Motherboard</td><td>@aa.lshw.base.motherboard</td>
+        </tr>
+        <tr>
+            <td>Firmware</td><td>@aa.lshw.base.firmware</td>
```

Comment:
its not clear if this is firmware for mobo, ipmi module, bios, or some other bit. Can you clarify, maybe in a `title=""`?

Reply:
I agree, it's not very descriptive. 

I might play around with this a bit more:

![image](https://user-images.githubusercontent.com/5204426/30447017-c86f2a7c-9958-11e7-8ce3-3451095cf4ee.png)